style2.css

```diff
@@ -0,0 +1,403 @@
+@media screen and (max-width: 1680px) {
+  body {
+      width: 100%;
+      height: 100%;
+  }
+}
+
+@media screen and (max-width: 1440px) {
+  .main__pic {
+      margin: -100px 0 0 0;
+  }
+}
+
+@media screen and (max-width: 1120px) {
+  .header__menu-link {
+      font-size: 14px;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/tesla394/pullRequest?prid=5642156)